### PR TITLE
Update 03_create_cluster.mdx

### PR DESCRIPTION
**Pull request by amrita-suresh**
Edit content for networking

## What Changed?

edit content for networking

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [ ] This PR adds new content
- [x] This PR changes existing content
- [ ] This PR removes existing content

### CODE REVIEW

File: product_docs/docs/edbcloud/beta/getting_started/03_create_cluster.mdx [hunk 1]
```diff
@@ -22,12 +22,11 @@ To create a cluster:
 10. Select the **Instance Type** based on the number of vCPUs and amount of memory desired.
 11. Select the **Storage** type and amount of storage needed for your cluster.
    !!! Note
-       Azure premium storage is the only option for the Preview.
+       Azure premium storage is the only option.
```
Comment (drothery-edb) [inline product_docs/docs/edbcloud/beta/getting_started/03_create_cluster.mdx:25]: Kelly and I think you should keep "For the Preview" unless it is not changing for GA.

**Comment by drothery-edb** [comment]
This will likely have a merge conflict. I will reach out to you Amrita.

File: product_docs/docs/edbcloud/beta/getting_started/03_create_cluster.mdx (hunk 1 above)
Comment (amrita-suresh) [reply product_docs/docs/edbcloud/beta/getting_started/03_create_cluster.mdx:25]: @aswright491 suggested dropping the "for the preview."

Comment (aswright491) [reply product_docs/docs/edbcloud/beta/getting_started/03_create_cluster.mdx:25]: It might change for GA.

**Review by kelpoole — APPROVED** [review]
Changes have been reviewed and minor updates applied

**Review by aswright491 — COMMENTED** [review]
this looks good to me.

**Pull request merged**